CONTRIBUTING.md

```diff
@@ -28,6 +28,17 @@ from the repository root. No additional configuration should be needed (see the
 [black documentation](https://black.readthedocs.io/en/stable/installation_and_usage.html#usage)
 for advanced usage).
 
+Docstring formatting: Although we are not extremely
+picky about docstrings,
+we do recommend [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) docstrings. 
+To make sure documentation is rendered correctly, we require that every `__init__`
+function contain an "Args" block. 
```

Comment:
```suggestion
function contains an "Args" block. 
```

Reply:
It's subjunctive, technically acceptable either way but makes me sound old: https://forum.wordreference.com/threads/subjunctive-after-require-he-requires-that-she-prepare-the-documents.230559/ so let's go with "contains" :D

Reply:
fwiw I would have gone with "contain" as well. And yes, I am old.

Reply:
oh and we are picky alright: https://github.com/pytorch/botorch/pull/1319#discussion_r923909707